client/lib/plans/constants.js

```diff
@@ -184,7 +184,17 @@ const getPlanPersonalDetails = () => ( {
 				},
 			}
 		),
-	getFeatures: () => [
+	// All plan features, used for checking plan abilities
+	getFeatures: () =>
+		uniq( [
+			...getPlanPersonalDetails().getDefaultPlanListFeatures(),
+			...getPlanPersonalDetails().getSignupFeatures(),
+			...getPlanPersonalDetails().getBlogSignupFeatures(),
+			...getPlanPersonalDetails().getPortfolioSignupFeatures(),
```

Comment:
Can you use `...this.getDefaultPlanListFeatures()` instead? That would be more readable than this.